bundles/org.eclipse.e4.ui.workbench/src/org/eclipse/e4/ui/workbench/UIEvents.java

```diff
@@ -310,6 +310,13 @@ public interface UILifeCycle {
 		 */
 		String PERSPECTIVE_RESET = TOPIC + TOPIC_SEP + "perspReset"; //$NON-NLS-1$
 
+		/**
+		 * Sent when a perspective is switched
+		 *
+		 * @since ???
```

Comment:
Must be the x.y of next release for this bundle. As it's a new API, bundle version must become `x.y+1.0`.
So please change bundle version to `1.14.0` in MANIFEST.MF, and set this to `@since 1.14`

Reply:
Done